datahub-web-react/src/app/entity/user/UserGroups.tsx

```diff
@@ -1,80 +1,206 @@
+                                <GroupItemColumn xl={8} lg={8} md={8} sm={12} xs={24}>
+                                    <Link to={entityRegistry.getEntityUrl(EntityType.CorpGroup, item.urn)}>
+                                        <GroupItem>
+                                            <Row style={{ padding: '9px 11px 9px 11px' }}>
+                                                <GroupTitle>{item.name}</GroupTitle>
+                                                <GroupMember>10 members</GroupMember>
+                                            </Row>
+                                            <Row style={{ padding: '2px 13px' }}>
+                                                <GroupDescription>
+                                                    {item.info?.description && item.info?.description.length > 85 ? (
+                                                        <Tooltip title={item.info?.description}>
+                                                            {`${item.info?.description.slice(0, 80)} ...`}
+                                                        </Tooltip>
+                                                    ) : (
+                                                        item.info?.description
+                                                    )}
+                                                </GroupDescription>
+                                            </Row>
+                                        </GroupItem>
+                                    </Link>
+                                </GroupItemColumn>
+                            );
+                        })}
+                    {/* dummy */}
+                    <GroupItemColumn xl={8} lg={8} md={8} sm={12} xs={24}>
+                        <GroupItem>
+                            <Row style={{ padding: '9px 11px 9px 11px' }}>
+                                <GroupTitle>Some title</GroupTitle>
+                                <GroupMember>10 members</GroupMember>
+                            </Row>
+                            <Row style={{ padding: '2px 13px' }}>
+                                <GroupDescription>
+                                    {DummyGroupDescriptionText.length > 85 ? (
+                                        <Tooltip title={DummyGroupDescriptionText}>
+                                            {`${DummyGroupDescriptionText.slice(0, 80)}...`}
+                                        </Tooltip>
+                                    ) : (
+                                        DummyGroupDescriptionText
+                                    )}
+                                </GroupDescription>
+                            </Row>
+                        </GroupItem>
+                    </GroupItemColumn>
+                    <GroupItemColumn xl={8} lg={8} md={8} sm={12} xs={24}>
+                        <GroupItem>
+                            <Row style={{ padding: '9px 11px 9px 11px' }}>
```

Comment:
We generally prefer to keep styling out of the JSX -- Use styled component above instead!